inference-engine/src/vpu/graph_transformer/src/stages/mvn.cpp

```diff
@@ -88,11 +90,13 @@ void FrontEnd::parseMVN(const Model& model, const ie::CNNLayerPtr& layer, const
     for (int i = 0; i < indicesSize; i++) {
         axes.insert(getDimFromAxis(ndims, indicesPtr[i]));
     }
+    const auto width = axes.count(Dim::W);
 
-    VPU_THROW_UNLESS(!axes.count(Dim::N) && axes.count(Dim::H) && axes.count(Dim::W),
```

Comment:
Why did you remove check for `Dim::H`?

Reply:
Because now if `axes.count(Dim::H)` is 0, it means there is only `Dim::W`, and this pr adds support for this case, and if `axes.count(Dim::H)` is 1, there is both  `Dim::W` and  `Dim::H`, whitch is already supported, so there is no need to check for it.

Reply:
Okay, thanks